package.json

```diff
@@ -47,6 +48,7 @@
     "native-dns-packet": "^0.1.1",
     "nconf": "~0.6.9",
     "properties": "^1.2.1",
+    "redis": "0.12.x",
```

Comment:
So why are there two redis dependencies? Do we need both of them? What do they do?